Ziti.NET.Standard/src/OpenZiti/ExtensionMethods.cs

```diff
@@ -38,6 +38,10 @@ public static string GetDescription(this Enum enumVal)
         {
             var type = enumVal.GetType();
             var memberInfo = type.GetMember(enumVal.ToString());
+            if (memberInfo == null || memberInfo.Length == 0)
+			{
+                return NO_DESC;

```

Comment:
I don't think we should plaster over this and I'd like to unwind this. if the description isn't returned we need to know what description and why. i don't think we should commit this change